enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,359 @@
+    subnets:
+    - subnet-1
+    - subnet-2
+    - subnet-3
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```  
+2. The installer will create the cluster ingress config object as follows:
+```yaml
+ oc edit ingress.config/cluster -o yaml
+ # Please edit the object below. Lines beginning with a '#' will be ignored,
+ # and an empty file will abort the edit. If an error occurs while saving this file will be
+ # reopened with the relevant failures.
+ #
+ apiVersion: config.openshift.io/v1
+ kind: Ingress
+ metadata:
+   creationTimestamp: "2022-08-30T01:03:17Z"
+   generation: 11
+   name: cluster
+   resourceVersion: "60888"
+   uid: de89cc95-449e-4ae5-a837-f4171fe61e36
+ spec:
+   domain: apps.testnlbapi.devcluster.openshift.com
+   loadBalancer:
+     platform:
+       aws:
+         type: NLB
+           networkLoadBalancer:
+             eip-allocations: eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>
+       type: AWS
+ status:
+   componentRoutes:
+```
+3. Ingress Operator will create the default ingress controller with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`.
+
+#### Create an Ingress Controller using Ingress Controller CR and assign the EIP allocation mentioned in the Ingress Controller CR.
+1. Cluster administrator creates an Ingress Controller:
+```yaml
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: null
+  name: test
+  namespace: openshift-ingress-operator
+spec:
+  endpointPublishingStrategy:
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: NLB
+          networkLoadBalancer:
+            eip-allocations: eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>,eipalloc-<redacted>
+    type: LoadBalancerService
+```
+
+Ingress Operator will create an NLB Ingress Controller called `test` with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+populated with the value from the eip-allocations field.
+
+#### Cluster administrator adds eip-allocations field the existing ingress controller
+
+Ingress Operator will create an NLB Ingress Controller with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+populated with the value from the eip-allocations field.
+
+#### Cluster administrator updates eip-allocations field the existing ingress controller
+
+Ingress Operator will update an NLB Ingress Controller with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+populated with the value from the eip-allocations field.
+
+### API Extensions
```

Comment:
Can you put this prior to Workflow descriptions, so the reader understands what is being changed?

Reply:
I did not get it. However the text has been changed now.

Reply:
Please see https://github.com/openshift/enhancements/pull/1593/files#r1581285577 and let me know if you need further instructions.

Reply:
I have copied API extensions above Workflow descriptions